ledger-core/inc/core/database/DatabaseSessionPool.hpp

```diff
@@ -75,20 +75,48 @@ namespace ledger {
                 soci::transaction tr(sql);
                 Migration<T::CURRENT_VERSION, T>::forward(sql, version);
 
+                // register rollback migration
+                _rollbackMigrations.emplace_back([](soci::session& sql) {
+                    int version = getDatabaseMigrationVersion<T>(sql);
```

Comment:
I missed this one; can you put also the `const` qualifier here ?

Reply:
You missed several actually. I’ll annotate them all.